lab-adam/lib/parse-body.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function(req, res, callback) {
+  var body = '';
+  req.on('data', function(data){
+    body += data.toString();
+    // res.write(req.body);
+  });
+  res.on('error', function(err) {
+    console.log(err);
```

Comment:
You should be using console.error for error, and it would make more sense to check the request for an error, not a response... because we don't receive a response on the server side, and thus will not have an error. :)